provider/service.py

```diff
@@ -120,6 +131,12 @@ def run(self):
 
             logging.debug("[changes] I made it out of the changes loop!")
 
+    def __isTriggerDocAssignedToMe(self, doc):
+        if "worker" in doc:
+            return doc["worker"] == self.workerId
+        else:
+            return self.workerId == "worker0"
```

Comment:
after speaking with @jasonpet this is the appropriate behavior. removed TODO comment